### DISCUSSION TITLE
Only root span showing for Jetty based application

### DISCUSSION THREAD

**Discussion by 1davidmichael** [grafana/grafana-opentelemetry-java#731] [Q&A] [answered]
Hi, I have a Java Jetty based application that I am trying to instrument with the Grafana OTEL agent. It is kind of working but I'm only getting a single root span and I'd expect to see a lot more. Is it possible I need to include additional flags or jars for proper instrumentation?
Here is what I have for OTEL flags currently:

```bash
-Denvironment=test-otel-agent
-Dservername=sq-test-otel-agent-4ea9d0a/health-76b772e-75676cdd68-rtxdc
-Dotel.resource.attributes=service.name=your-service-name=health
-Dotel.resource.attributes=deployment.environment=test-otel-agent
-Dotel.resource.attributes="kubernetes.pod=health-76b772e-75676cdd68-rtxdc,kubernetes.namespace=sq-test-otel-agent-4ea9d0a"
-Dotel.exporter.otlp.protocol=http/protobuf
-Dotel.instrumentation.jetty.enabled=true
-Dotel.instrumentation.apache-httpclient.enabled=true
-Dotel.exporter.otlp.endpoint=https://otel-alloy.qa.k8s.use1/.*****.com
```

Here is an example of what I see:
![image](https://github.com/user-attachments/assets/717117b6-8a95-40d0-82b7-a2c961006a48)

**Comment by zeitlinger** [comment]
I'm not sure what you are missing - but you can try https://opentelemetry.io/docs/zero-code/java/agent/disable/#suppressing-controller-andor-view-spans

**Reply by 1davidmichael** [reply, marked as answer]
hi @zeitlinger. I think this was a mistake in my understanding of how the application is functioning. We enabled additional tracing of methods via the `otel.instrumentation.methods.include` config option and we now see more details. This can be closed. Thanks.
